.openpublishing.redirection.json

```diff
@@ -101,6 +101,10 @@
         "redirect_url": "/office/dev/add-ins/concepts/add-in-development-best-practices", 
         "redirect_document_id": true 
       }, 
+      {   
+        "source_path": "/office/dev/add-ins/concepts/web-viewers-used-by-office-web-add-ins.md", 
```

Comment:
Sorry, one more comment -- seems like source path should be:

`docs/concepts/web-viewers-used-by-office-web-add-ins.md`